substratevm/src/org.graalvm.polyglot.nativeapi/src/org/graalvm/polyglot/nativeapi/types/PolyglotNativeAPITypes.java

```diff
@@ -119,13 +116,22 @@
 
     }
 
-    @CPointerTo(nameOfCType = "poly_handle")
     @CTypedef(name = "poly_handle")
     public interface PolyglotHandle extends PointerBase, ObjectHandle {
 
     }
 
-    @CPointerTo(nameOfCType = "poly_reference")
```

Comment:
@pejovica what should we change to make this possible? You were working on TypeDefs recently.